src/Identity/IdentityServer/BaseRequestValidator.cs

```diff
@@ -133,6 +142,15 @@ public abstract class BaseRequestValidator<T> where T : class
             var verified = await VerifyTwoFactor(user, twoFactorOrganization,
                 twoFactorProviderType, twoFactorToken);
 
+            var isOtpCached = Core.Utilities.DistributedCacheExtensions.TryGetValue(_distributedCache, user.Email, out string cachedToken);
```

Comment:
⛏️ You can eliminate the unused output variable with just `_`.

Reply:
Updated to use `_`.

Reply:
Was this undone?